code/modules/wiremod/components/bci/user_status.dm

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
This should really just be merged into the health component, not sure why it needs to be a BCI exclusive either.

Reply:
I don't think it's that big of a deal, actually.

Reply:
> I feel like the health scanner component makes more sense as a entity generic scanner for health, as I don't know if there is a species limitation for what can be unconscious/asleep/etc. Meanwhile if this component is reading the 'user' state and it's only for a BCI shell, then it makes a lot of sense that an internal circuit have access to read the user's mind more than a external health scanner.

Consciousness on the mob-level
It's specifically on the `stat` variable and it has 4 stages.
CONSCIOUS -> SOFT_CRIT -> UNCONSCIOUS -> HARD_CRIT -> DEAD